### ISSUE TITLE
Accordion/useAccordionContext

### ISSUE THREAD

**Issue by grandalf6** [reach/reach-ui#728]
### Current Behavior

While using `Accordion` and `useAccordionContext` inside child of Accordion, I am not able to receive correct expected value of `openPanels`.

Here is code:
```jsx
const App = () => (
  <div className="App">
    <Accordion>
      {[0, 1, 2].map((elem) => (
        <Child index={elem} />
      ))}
    </Accordion>
  </div>
);

const Child = ({ index }) => {
  const { openPanels } = useAccordionContext();
  // openPanels.length is always 0
  console.log(openPanels);

  return (
    <AccordionItem>
      <AccordionButton>{index}</AccordionButton>
      <AccordionPanel>{index} content</AccordionPanel>
    </AccordionItem>
  );
};
```

The output from console:
```jsx
[]
```

### Expected behavior

I expect console.log to be:
```jsx
[0]
```

### Reproducible example

[CodeSandbox link](https://codesandbox.io/s/quiet-snowflake-ygdp2?file=/src/index.js)

### Suggested solution(s)

None

### Additional context

None

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Reach Package    |    @reach/accordion     |    0.12.1     |
| React            |     NA    |    16.12.0     |
| Browser          |     Google Chrome    |     Version 87.0.4280.141    |
| Assistive tech   |    none     |     NA    |
| Node             |    NA     |     v10.20.1    |
| npm/yarn         |     NA (codesandbox)  |     NA    |
| Operating System |     Windows    |    10 Pro  |

**Comment by chaance** [comment]
Fixed in 0.13.0

**Comment by jwlms** [comment]
Any word on when this fix will be released?
